app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -55,11 +60,49 @@ export default function NoteTable({
                 />
               </td>
               <td>
-                <div className="flex flex-row items-center justify-start text-gray-900">
+                <div className="flex flex-row items-center justify-start">
                   {note.title}
                 </div>
               </td>
-              <td>{note.description}</td>
+              <td>
+                <div className="flex flex-row items-center justify-start">
+                  {note.description}
+                </div>
```

Comment:
Please truncate the note description here.
Please refer [Text Overflow](https://tailwindcss.com/docs/text-overflow) from tailwindCSS for more details.